.gitignore

```diff
@@ -0,0 +1 @@
+task/source-build/**/scripts/
```

Comment:
why is this needed?

Reply:
It's hard to edit the inline Python code inside the YAML file, so I edit the code by creating a file under this directory, then copy/paste back to the YAML.

Reply:
so this is  just temporary right? so it shouldn't be in the PR, but only in your repo

Reply:
This script should not be included in this pr. I'm using it to edit the script outside the task.

Reply:
I think it should stay in your local repository, because different developers have own habits.

Reply:
Mark this as resolved since the code has been updated and a new story was opened for moving to a separate git repo.